doc/chapter-firststeps.asciidoc

```diff
@@ -24,127 +25,135 @@ usage: ./newsboat [-i <file>|-e] [-u <urlfile>] [-c <cachefile>] [-x <command> .
         --cleanup                   remove unreferenced items from cache
 ----
 
-This means that Newsboat can't start without any configured feeds. To add
-feeds to Newsboat, you can either add URLs to the configuration file
-_~/.newsboat/urls_ or you can import an OPML file by running `newsboat -i
-blogroll.opml`. To manually add URLs, open the file with your favorite text
-editor and add the URLs, one per line:
+This means that Newsboat can't start without any configured feeds.
+
+=== Adding Feeds
+
+To add feeds to Newsboat, you can simply add one feed URL per line to the _~/.newsboat/urls_
+configuration file:
 
 	http://rss.cnn.com/rss/cnn_topstories.rss
 	http://newsrss.bbc.co.uk/rss/newsonline_world_edition/front_page/rss.xml
 
-If you need to add URLs that have restricted access via username/password, simply
-provide the username/password in the following way:
+You can also import an OPML file by running `newsboat -i blogroll.opml`
+
+
+****
+*Adding comments* Lines that start with `#` can contain anything you want.
+Comments are ignored by Newsboat, but can serve as documentation for you.
+Please note, that commenting out URLs for debugging
+purposes might lead to unexpected data loss, see
+<<cleanup-on-quit,`cleanup-on-quit`>> for more details.
+****
+
+*Feeds with restricted access*
```

Comment:
Yeah me too actually! I used headings at first but then switched to bold text because every heading is styled the same way on the man page. Bold text looks like an indented heading on the man page.
I don't know if the man page is priority here? If not, I'm happy to switch back to sub headings.

Reply:
I wouldn't say that either the manpage or the HTML are a priority; I'd prefer both to be useful :)

I guess in the long run, we should split the info such that manpage is the reference (mostly lists and tables), and HTML is more of a tutorial (with prose and more explanations). Then we can tailor headings and such to the medium—less nesting in the manpage, more nesting in HTML.

But that's for another PR. Let's keep bold text here for now.